deploy/logstash-operator/templates/crd-logstash-deployment.yaml

```diff
@@ -50,6 +50,23 @@ spec:
                         container
                       type: string
                       default: /usr/share/GeoIP
+                metrics:
```

Comment:
Those properties need to be defined according to whats needed for the exporter

Reply:
I would expect something like this:

metrics: 
  enabled: true/false
  option1: ...
  ...